src/icon/aloneInTheBackIcon.tsx

```diff
@@ -0,0 +1,18 @@
+// tslint:disable:max-line-length
+import React from 'react'
+import BaseIcon from '_utils/icon'
+import { BaseIconDefaultProps } from '_utils/icon/BaseIcon'
+
+export const AloneInTheBackIcon = (props: Icon) => (
+  <BaseIcon {...props}>
```

Comment:
Should we make `statusIcon` for more flexibility so we can have the red strikethrough?

Reply:
I guess it's not required since I copied ComfortIcon since it's supposed to be used in the same context.